src/components/search/curated/[slug]/index.tsx

```diff
@@ -129,21 +226,29 @@ const CuratedTopic: React.FC<CuratedTopicProps> = ({topic, topicData}) => {
             </div>
           </div>
         )}
-        <div className="sm:px-5 px-3">
+        <div className="sm:px-5 px-3 sm:pt-16 pt-8">
           {!isEmpty(sections) &&
-            sections.map((section: any) => {
-              return (
-                <section className="pb-16" key={section.title}>
-                  {!section.image && !section.description ? (
-                    // simple section
-                    <div className="flex w-full pb-6 items-center justify-between">
-                      <h2 className="lg:text-2xl sm:text-xl text-lg dark:text-white font-semibold leading-tight">
-                        {section.title}
-                      </h2>
-                    </div>
-                  ) : (
+            sections.map((section: any, i: number) => {
+              switch (true) {
+                case !section.image && !section.description:
+                  return (
+                    <section className="pb-16" key={section.title}>
+                      <div className="flex w-full pb-6 items-center justify-start">
+                        <h2 className="lg:text-2xl sm:text-xl text-lg dark:text-white font-semibold leading-tight">
+                          {section.title}
+                        </h2>
+                      </div>
+                      <DynamicCardGrid
+                        section={section}
+                        completedCoursesIds={completedCoursesIds}
+                        location={location}
+                      />
+                    </section>
+                  )
+                case section.resources.length === 2:
```

Comment:
using a switch case to add special cases we want to handle. 

Right now we want to handle sections:
- that dont have a description or image 
- that have 2 resources
- that have 4 resources and no image

everything else will continue to be default behavior down below